src/PrestaShopBundle/Resources/config/services/bundle/form/form_type.yml

```diff
@@ -1874,10 +1874,13 @@ services:
     tags:
       - { name: form.type }
 
-  form.type.sell.product.combination.combination_manager_type:
-    class: 'PrestaShopBundle\Form\Admin\Sell\Product\Combination\CombinationManagerType'
-    parent: 'form.type.translatable.aware'
-    public: true
+  PrestaShopBundle\Form\Admin\Sell\Product\Combination\CombinationManagerType:
+    public: false
+    autowire: true
+    autoconfigure: true
```

Comment:
wasn't mandatory change, but this way it will be easier to integrate in develop.
related PR in develop: https://github.com/PrestaShop/PrestaShop/pull/31138/files